src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -124,8 +124,7 @@
 # import job steps from other toil pipelines
 from toil_scripts.adam_pipeline.adam_preprocessing import * #static_adam_preprocessing_dag
 from toil_scripts.bwa_alignment.bwa_alignment import * #download_shared_files
-from toil_scripts.gatk_germline.germline import * #batch_start
-from toil_scripts.gatk_processing.gatk_preprocessing import * #download_gatk_files
```

Comment:
Just to confirm here, we removed this import because `toil_scripts.gatk_processing.gatk_preprocessing` moved into `toil_scripts.gatk_germline.germline`, right?